apps/beeswax/src/beeswax/models.py

```diff
@@ -64,7 +64,7 @@ class STATE(Enum):
                  (librdbms_dbms.MYSQL, 'MySQL'), (librdbms_dbms.POSTGRESQL, 'PostgreSQL'),
                  (librdbms_dbms.SQLITE, 'sqlite'), (librdbms_dbms.ORACLE, 'oracle'))
 
-  owner = models.ForeignKey(User, db_index=True)
+  owner = models.ForeignKey(User, on_delete=models.CASCADE, db_index=True)
```

Comment:
We could need to generate new db migration files if we updates the models.

Also, could you explain what was the warning, do we need to be explicit now? Was the default not 'CASCADE'?

Reply:
warning is `on_delete will be a required arg for ForeignKey in Django 2.0. Set it to
models.CASCADE on models`
and the link is https://docs.djangoproject.com/en/2.0/ref/models/fields/#django.db.models.ForeignKey.on_delete